### ISSUE TITLE
Issues with the R code

### ISSUE THREAD

**Issue by matteodefelice** [ejlchappin/emlab-generation2#9]
Firstly, thanks for sharing this code. I was able to run everything with Netbeans 12 but the documentation for the analysis seems to be outdated, isn't it?

I have created my config following `example_config.R` and including the results' folder. Then, the RMarkdown notebook doesn't compile because the script `emlab/init.R` looks for a `emlab_results_meta_directory` which is not defined anywhere. The part of the script not running is the "Read description file", if I comment this part the notebook seems to work!

**Comment by marcmelliger** [comment]
Just add a new variable to config.R (similar to emlab_results_directory): 
`emlab_results_meta_directory <- "[...]/emlab-generation2/results_meta"` (adjust the path to your directory)
 Then create this folder in the emlan-generation2 folder.

Next version should have this already set up.

**Comment by marcmelliger** [comment]
Also, yes. The documentation is not up to date, currently. I'm trying to add the most important points in the comming months

**Comment by matteodefelice** [comment]
Thanks a lot, however it is not working, the notebook looks for a `main.R` that doesn't exist (any more) and the Shiny app looks for a `scenario_description_initial_name` which is not defined any where. However, the R code is secondary...the CSV file produced as outputs are clear enough to be analysed in any case. However, I will check for any new commit to this repository!

**Comment by marcmelliger** [comment]
Yeah, sorry. The example notebooks in this commit here are outdated. We'll update these analysis scripts soon to something useable.
